coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -20,14 +20,19 @@ def __init__(self, documentation, docstyle_definition,
         """
         Instantiates a new DocumentationComment.
 
-        :param documentation: The documentation text.
-        :param language:      The language of the documention.
-        :param docstyle:      The docstyle used in the documentation.
-        :param indent:        The string of indentation used in front
-                              of the first marker of the documentation.
-        :param marker:        The three-element tuple with marker strings,
-                              that identified this documentation comment.
-        :param range:         The position range of type TextRange.
+        :param documentation:
+            The documentation text.
+        :param docstyle_definition:
+            The ``DocstyleDefinition`` instance that defines what docstyle is
+            being used in the documentation.
+        :param indent:
+            The string of indentation used in front of the first marker of the
+            documentation.
+        :param marker:
+            The three-element tuple with marker strings, that identified this
+            documentation comment.
+        :param range:
+            The position range of type TextRange.
```

Comment:
Just quickly put `TextRange` also in double back ticks like `DocstyleDefinition` and we are good to merge this :)